recipes/spleeter-gpu/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "spleeter-gpu" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 49e776f26c561357eb9608a459030ab46cfe11edf21fb17470630a0d3777d453
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: noarch
```

Comment:
```suggestion
  noarch: python
```

Reply:
🤦‍♂ ... sorry

Reply:
Got `noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python.` from the linter ? Is it related to Python version constraint selectors ?

Reply:
Yes, you'll have to include `importlib_resources` as a dependency on all platforms rather than using a selector but it's available for all platforms so I think it's worth it for the added simplicity.

Reply:
Perfect !